Chawki-Mbarki/Portfolio1
# Portfolio: Improve accessibility and apply html best practices

👋 Hi,
This PR improves the accessibility of the portfolio website. It also fixes some validation errors and warnings. 👍

## The main changes in this PR are:

  - Changed the headings hierarchy to make it more logical and meaningful. For example, I changed the project titles from `h2` to `h3` and the about section title from `h1` to `h2.` 🧠
  - Added `alt` attributes to all images, with descriptive text that explains their content or function. 🖼️
  - Linked `labels` to form `inputs` using the `for` attribute. 🏷️
  - Added `title` attributes to anchor tags, to provide additional information on hover. 🔗
  - Renamed some `SVG` files to make them more organized and avoid confusion. 🗂️
  - Used semantic `HTML` elements, such as `section,` `article,` and `footer,` where appropriate. 📃
  - Fixed some `HTML` validation errors and warnings, such as duplicate ids, missing end tags, and obsolete attributes. 🚫

## Testing? 🧪
  - To test this PR, you can use various tools and techniques to check the accessibility and HTML quality of the website. For example:
    - Use a screen reader or a browser extension like ChromeVox to navigate the website and see how it sounds. 🔊
    - Use a tool like Lighthouse or WebAIM WAVE to audit the website and get a report on its accessibility score and issues. 📊
    - Use a tool like HTML Validator or W3C Markup Validation Service to validate the HTML code and find any errors or warnings. ✅

This PR is part of the Microverse curriculum project. Feedback is welcome and appreciated. 😊
